### DISCUSSION TITLE
I can't login to my Hub Cloud server

### DISCUSSION THREAD

**Discussion by Dreamfullstacker** [mozilla/hubs#5494] [Hubs Cloud Questions] [answered]
I have deployed Hub Cloud on AWS
For that, I have created .com and .link
Using that I have deployed stack
I can check the domain https://stealoo-cinema.com/
But I want to customize some pages
For that, I forked the https://github.com/mozilla/hubs
and then clone the repo on my local
after install module using npm ci
I can't login "npm run login"
about the HOST, I have inputed stealoo-cinema.com
But it said like this
"Sorry, that doesn't look like a Hubs Cloud server."
How can I handle this problem

**Comment by matthewbcool** [comment, marked as answer]
Were you able to resolve this issue @Dreamfullstacker? Why was npm run login not recognizing your domain?

**Reply by Dreamfullstacker** [reply]
Sorry everyone
Long time ago, I have been blocked some IP address for my project security test.
After that, I forgot repair that.
I reinstalled system and then recognized that.
Now the terminal working well
Thank you for your care
